docs/reference/compose_alpha_publish.md

```diff
@@ -5,10 +5,11 @@ Publish compose application
 
 ### Options
 
-| Name                      | Type | Default | Description                     |
-|:--------------------------|:-----|:--------|:--------------------------------|
-| `--dry-run`               |      |         | Execute command in dry run mode |
-| `--resolve-image-digests` |      |         | Pin image tags to digests.      |
+| Name                      | Type     | Default | Description                                                           |
+|:--------------------------|:---------|:--------|:----------------------------------------------------------------------|
+| `--dry-run`               |          |         | Execute command in dry run mode                                       |
+| `--oci-version`           | `string` |         | OCI Image specification version (automatically determined by default) |
```

Comment:
OCI artifact specification

Reply:
OCI artifacts are dead, long live OCI images: https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidelines-for-artifact-usage

😬

Reply:
really? So this is an non-secret plan to make all this think damn confusing as an "image" can now either be an image or an arbitrary artifact 🥸